benefits/eligibility/templates/eligibility/confirm.html

```diff
@@ -1,6 +1,31 @@
-{% extends "core/page.html" %}
+{% extends "core/base.html" %}
+
+{% block page_title %}
+  {{ page.title }} |
+{% endblock page_title %}
 
 {% block main_content %}
-  <div class="container">{% include "core/includes/previous-page-link.html" %}</div>
-  {{ block.super }}
+  <div class="container">
+    {% include "core/includes/previous-page-link.html" %}
+
+    <div class="row justify-content-center">
+      <div class="col-lg-10">
+        {% block icon_title %}
```

Comment:
Just a comment: since this now inherits from `base.html`, there's really no need for these `{% block %}` sections -- they are meant to create (or use) placeholders in a parent template.

We could just have the content inside the blocks directly here:

* `{% block icon_title %}`
* `{% block paragraph_content %}`
* `{% block forms %}`

And same comment for `eligibility/index.html`

Reply:
Ended up keeping {% block paragraph_content %} and {% block forms %} and modifying how the nav_buttons work cos it was messing up the width of the page

Reply:
Just curious, why keep those two `block`?

Reply:
Forms has to be `block` cos I'm overriding both form and the CTA

I opted to put paragraph content in `block` because I wanted to be able to set their own padding for the paragraphs from the inside. 

I don't want to set padding on `paragraph_content` at the `base.html` level, because in the cases where there is a page with a _blank_ `paragraph_content`, it will still add padding. And we don't want that.

Reply:
cc @angela-tran @thekaveman 

I need to override `paragraph_content` b/c:

1. I need to add these classes `pt-4 pb-4 pb-lg-8` which set padding-bottom to 64px, among other things, from the *inside* of the parent div.

2. If we add `pt-4 pb-4 pb-lg-8` to the main parent itself, when there are pages without `paragraph_content` (which most pages don't have `paragraph_content`), the pages will have a giant 64px blank space. Padding applies even if the div inside is empty. We should not be adding padding classes to divs that might be empty/un-used in the base template. 

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/3673236/195183538-8750beef-5080-417c-8bf2-6c3f576102e1.png">

Reply:
Ok I don't want any of this to be blocking for this PR since it really doesn't matter to the end result...

> Forms has to be `block` cos I'm overriding both form and the CTA

There isn't anything to override. `{% block %}` is used to place content in specific spot (the block) defined in a **parent** template. The parent here `base.html` does not define a `{% block forms %}`, so there is no spot to place/override anything. In `confirm.html`, this block turns into... exactly nothing in the rendered HTML.

By using `{% block forms %}` here, it creates a block that another template, inheriting from `confirm.html`, could fill in; that doesn't really make sense / isn't our use-case for these form pages.

> I opted to put paragraph content in `block` because I wanted to be able to set their own padding for the paragraphs from the inside... I don't want to set padding on paragraph_content at the base.html level

So here it looks like a change to put the paragraphs into `{% block explanatory-text %}` which makes sense - the `base.html` defines this, so this places the paragraphs in that spot in the base template.